DQMOffline/Trigger/python/PhotonMonitor_cff.py

```diff
@@ -0,0 +1,47 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.PhotonMonitor_cfi import hltPhotonmonitoring
+
+#HLT_SinglePhoton200_IDTight
+SinglePhoton300_monitoring = hltPhotonmonitoring.clone()
+SinglePhoton300_monitoring.FolderName = cms.string('HLT/Photon/Photon300/')
+SinglePhoton300_monitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Photon300_NoHE_v")
+
+
+# HLT_SinglePhoton175_IDTight
+SinglePhoton175_monitoring = hltPhotonmonitoring.clone()
+SinglePhoton175_monitoring.FolderName = cms.string('HLT/Photon/Photon175/')
+SinglePhoton175_monitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Photon175_v*")
```

Comment:
there is photonselection in the python file photonMonitoring_cfi, which will be created from .cc file.
 desc.add<std::string>("photonSelection", "pt > 145 && eta<1.4442 && hadTowOverEm<0.0597 && full5x5_sigmaIetaIeta()<0.01031 && chargedHadronIso<1.295");